tools/src/tester/test.ts

```diff
@@ -7,22 +7,23 @@
 * compatible open source license.
 */
 
-import { LogLevel, Logger } from '../Logger'
+import {Logger, LogLevel} from '../Logger'
 import TestRunner from './TestRunner'
-import { Command, Option } from '@commander-js/extra-typings'
+import {Command, Option} from '@commander-js/extra-typings'
 import {
   get_opensearch_opts_from_cli,
   OPENSEARCH_INSECURE_OPTION,
   OPENSEARCH_PASSWORD_OPTION,
   OPENSEARCH_URL_OPTION,
-  OPENSEARCH_USERNAME_OPTION, OpenSearchHttpClient
+  OPENSEARCH_USERNAME_OPTION,
+  OpenSearchHttpClient
 } from '../OpenSearchHttpClient'
 import ChapterReader from './ChapterReader'
 import ChapterEvaluator from './ChapterEvaluator'
 import OperationLocator from './OperationLocator'
 import SchemaValidator from './SchemaValidator'
 import StoryEvaluator from './StoryEvaluator'
-import { ConsoleResultLogger } from './ResultLogger'
+import {ConsoleResultLogger} from './ResultLogger'
```

Comment:
I'm surprised the linter isn't complaining about this as it should be enforcing spacing around the braces (or at least it was)

Reply:
I setup my IDE to run `--fix` on save. I thought the linter wanted space too.

Reply:
This looks suspicious. Other files have a space. Something is off with the eslint rule file being used?

https://github.com/opensearch-project/opensearch-api-specification/blob/main/tools/src/coverage/coverage.ts#L12

Reply:
Fixed it. Didn't dig too deep but `object-curly-spacing` rule was deprecated in ESLint v8.53.0 and moved to a different plugin. Installing the plugin and enabled that rule solved it.